models/inviteModel.js

```diff
@@ -35,36 +35,38 @@ async function inviteUser(manager_id, email) {
 	}
 
 	// // Generate an inviteCode
-	 const inviteCode = randomString(16);
+	const inviteCode = randomString(16);
 
 	// Create an invite
 	const [invite] = await db('invites').insert(
 		{ manager_id, email, inviteCode },
 		'inviteCode'
 	);
 
-	//MailGun 
-	const mailgun = new Mailgun({apiKey: mailgunKey, domain: mailgunDomain});
-	
-	//Content of Email Being Sent
-	const data = {
-		from: `${manager_id}@well-broomed.com`,
-		to: email,
-		subject: "Well-Broomed Invitation",
-		html: "Hello! You have been invited to join a property management team on Well-Broomed." + 
-			  "If you would like to accept this invitation, please click this link: " + `${process.env.serverURL}/accept/` + inviteCode,
-	}
-
-	mailgun.messages().send(data, function (err, body) {
-		if (err) {
-			console.log("Mailgun got an error: ", err);
-			return {mailgunErr : err};
-		}
-		else
-			console.log('body:', body);
-	});
-
-	return { inviteCode: data };
+	// //MailGun
+	// const mailgun = new Mailgun({apiKey: mailgunKey, domain: mailgunDomain});
+
+	// //Content of Email Being Sent
+	// const data = {
+	// 	from: `${manager_id}@well-broomed.com`,
+	// 	to: email,
+	// 	subject: "Well-Broomed Invitation",
+	// 	html: "Hello! You have been invited to join a property management team on Well-Broomed." +
+	// 		  "If you would like to accept this invitation, please click this link: " + `${process.env.serverURL}/accept/` + inviteCode,
+	// }
+
+	// mailgun.messages().send(data, function (err, body) {
+	// 	if (err) {
+	// 		console.log("Mailgun got an error: ", err);
+	// 		return {mailgunErr : err};
+	// 	}
+	// 	else
+	// 		console.log('body:', body);
+	// });
+
+	// return { inviteCode: data };
+
+	return { inviteCode: invite };

```

Comment:
Is the mailgun action moved to a different place or just commented out for testing right now?

Reply:
Whoops. Yeah, it was commented out for testing. Nice catch.